Scheduler/observing_blocks_maintel/BLOCK-T87.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "BLOCK-T87",
+  "id": "b07a2321-d0e7-49d1-90e7-283d179845b7",
+  "program": "BLOCK-T87",
+  "constraints": [],
+  "scripts": [
+    {
+      "name": "maintel/offset_camera_hexapod.py",
+      "standard": true,
+      "parameters": {
+        "z": -1500,
+        "ignore": [
+          "MTMount",
```

Comment:
This has to be the component attribute name, so `mtmount`, `mthexapod_2`, etc...